src/components/Editor/ProjectComponentLoader/ProjectComponentLoader.js

```diff
@@ -10,21 +10,22 @@ const DEFAULT_PROJECT_TYPE = 'python'
 
 const ProjectComponentLoader = (props) => {
   const loading = useSelector((state) => state.editor.loading);
-  const { identifier, projectType } = useParams()
+  const { identifier } = useParams()
   const embedded = props.embedded || false;
   const user = useSelector((state) => state.auth.user)
   const accessToken = user ? user.access_token : null
+  const project = useSelector((state) => state.editor.project)
+  const projectType = project ? project.project_type : DEFAULT_PROJECT_TYPE
 
   useEmbeddedMode(embedded);
-  useProject(projectType || DEFAULT_PROJECT_TYPE, identifier, accessToken);
+  useProject(projectType, identifier, accessToken);
```

Comment:
I'm not quite sure about this part - are there any cases where `project` is actually set in state at this point? It may be worth refactoring `projectType` out of the `useProject` hook for now, as I suspect we're going to determine which default project to load up differently in the future anyway... 🤔

Reply:
Done in [bd18d87](https://github.com/RaspberryPiFoundation/editor-ui/pull/347/commits/bd18d878d8fc1609c3dc9fe1b8a2afe7bf6beb7f)